cdc/sinkv2/eventsink/mq/mq_dml_sink.go

```diff
@@ -101,7 +101,10 @@ func newSink(ctx context.Context,
 
 // WriteEvents writes events to the sink.
 // This is an asynchronously and thread-safe method.
-func (s *dmlSink) WriteEvents(rows ...*eventsink.RowChangeCallbackableEvent) error {
+func (s *dmlSink) WriteEvents(
```

Comment:
There is already a `Context` passed into `dmlSink` when creates it. Seems we should reuse that context.